packages/requestNetworkArtifacts/artifacts.json

```diff
@@ -41,6 +41,7 @@
         "last-requesterc20-0xd2d6158683aee4cc838067727209a0aaf4359de3": "RequestERC20/RequestERC20-0.2.7-bnty-main.json",
         "last-requesterc20-0x9f8f72aa9304c8b593d555f12ef6589cc3a579a2": "RequestERC20/RequestERC20-0.2.7-mkr-main.json",
         "last-requesterc20-0x8762db106b2c2a0bccb3a80d1ed41273552616e8": "RequestERC20/RequestERC20-0.2.8-rsr-main.json",
+        "last-requesterc20-0x6b175474e89094c44da98b954eedeac495271d0f": "RequestERC20/RequestERC20-0.2.8-dai-main.json",
```

Comment:
for consistency, we should keep the same order (it allows safe batch manipulation). I'm being picky :)